autonomi/src/client/quote.rs

```diff
@@ -104,7 +114,9 @@ impl Client {
             })
             .collect();
 
-        process_tasks_with_max_concurrency(futures, *FILE_UPLOAD_BATCH_SIZE).await
+        let parallism = std::cmp::min(*CHUNK_UPLOAD_BATCH_SIZE * 8, 128);
+
+        process_tasks_with_max_concurrency(futures, parallism).await
```

Comment:
Why are we ignoring the constant here and doing our own maths?

Reply:
Change was introduced here: https://github.com/maidsafe/autonomi/pull/2929

Not sure why we don't allow users to set it lower than 8?

Maybe @maqi can shed some light here

Reply:
a quoting task is supposed to be much lighter than the chunk_upload task.
hence got such correlation.